test/integration/narrow/routes/healthy.test.js

```diff
@@ -1,12 +1,13 @@
 describe('Healthy test', () => {
-  let server
-
   jest.mock('sequelize')
-  const createServer = require('../../../../server')
+
```

Comment:
We shoull avoid mocking something we don't own, like `sequelize`.

If the `database-service` mock is removed from the `__mock__`, folder then the `sequelize` mock can be removed, and the line 

`jest.mock('../../../../server/services/message-service')`

added after line 8 to get the same behaviour.

Reply:
Yup. I'll make the change.